executor/write_test.go

```diff
@@ -24,12 +24,19 @@ import (
 	"github.com/pingcap/tidb/executor"
 	"github.com/pingcap/tidb/kv"
 	"github.com/pingcap/tidb/model"
+	"github.com/pingcap/tidb/session"
 	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/store/mockstore"
 	"github.com/pingcap/tidb/table"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/testkit"
 )
 
+type testBypassSuite struct{}
```

Comment:
Why not reuse the `testSuite`?

Reply:
This test create a new store to set latch capacity, it's not common in `testSuite`.
And if I write it in `testSuite`, I found `TestShow` fail:

```
FAIL: show_test.go:35: testSuite.TestShow

show_test.go:141:
    result = tk.MustQuery(testSQL)
/media/genius/OS/project/src/github.com/pingcap/tidb/util/testkit/testkit.go:180:
    tk.c.Assert(errors.ErrorStack(err), check.Equals, "", comment)
... obtained string = "" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/store/tikv/snapshot.go:312: tikv aborts txn: leveldb: closed\n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/store/tikv/snapshot.go:314: \n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/store/tikv/snapshot.go:269: \n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/store/tikv/snapshot.go:218: \n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/kv/union_store.go:186: \n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/store/tikv/txn.go:96: \n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/structure/list.go:220: \n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/structure/list.go:164: \n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/meta/meta.go:499: \n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/meta/meta.go:512: \n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/util/admin/admin.go:52: \n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/ddl/stat.go:58: \n" +
...     "/media/genius/OS/project/src/github.com/pingcap/tidb/kv/txn.go:57: \n" +
```

So I just create a new `Suite`.